src/otx/algorithms/common/adapters/mmcv/utils/automatic_bs.py

```diff
@@ -79,6 +79,7 @@ def train_func_single_iter(batch_size):
         if idx_hooks_to_remove:
             idx_hooks_to_remove.sort()
             for i in reversed(idx_hooks_to_remove):
+                print(f"{copied_cfg.custom_hooks[i]} deleted.")
```

Comment:
Maybe to be deleted?

Reply:
Done. https://github.com/openvinotoolkit/training_extensions/pull/2510/commits/e1cf24fe04c36af2963f098052c248475a68e2b7

Reply:
I think he said this print statement can be deleted, is it right @goodsong81? And if you think this is necessary, I think `logger.debug` is better than `print`.

Reply:
removed it.